dev/src/Command/AddComponentCommand.php

```diff
@@ -111,17 +123,32 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 ->render();
         }
 
-        $productHomepage = $this->getHelper('question')->ask(
-            $input,
-            $output,
-            new Question('What is the product homepage? ')
-        );
-
-        $productDocumentation = $this->getHelper('question')->ask(
-            $input,
-            $output,
-            new Question('What is the product documentation URL? ')
-        );
+        $version = basename(dirname($proto));
+        // yaml file contains documentation_uri and
+        // it is available at the same location as proto file
+        $libraryName = strtolower($new->componentName);
+        $yamlFileName = $libraryName . '_' . $version . '.yaml';
+        $yamlFilePath = dirname($proto) . '/' . $yamlFileName;
```

Comment:
I think we can make this a bit more readable:
```suggestion
        $yamlFilePath = sprintf('%s/%s_%s.yaml', 
            dirname($proto),
            strtolower($new->componentName),
            $version 
        );
```

Reply:
Working on this..

Reply:
I see some issues with `loadProtoContent` function. If the `$proto` path there is `../googleapis/google/cloud/config/v1/config.proto`, that will go and check for the file `https://raw.githubusercontent.com/googleapis/googleapis/master/../googleapis/google/cloud/config/v1/config.proto` which is not the same uri as `https://raw.githubusercontent.com/googleapis/googleapis/master/google/cloud/config/v1/config.proto`

Reply:
Changed this in the logic a bit in this PR.